.env.example

```diff
@@ -0,0 +1,11 @@
+# Do not use the same values in the non-dev environments!
+
+MARIADB_ROOT_PASSWORD="8HrA2pzlmmgU3Avi"
+MARIADB_DATABASE="svitbo"
+MARIADB_USER="admin"
+MARIADB_PASSWORD="Q9bl907JI4y8"
+
+INITIAL_ADMIN_USERNAME="kirby"
```

Comment:
Why is it `kirby`? Maybe let's choose something more serious like `admin`?

Reply:
It's a security feature.

Reply:
Well, idk about that, but let it keep as it is.